src/ravijit.cpp

```diff
@@ -272,9 +272,17 @@ static const luaL_Reg ravilib[] = {{"iscompiled", ravi_is_compiled},
 LUAMOD_API int raviopen_llvmjit(lua_State *L) {
   luaL_newlib(L, ravilib);
   /* faster calls some maths functions */
+  #ifdef __cplusplus
```

Comment:
I don't think you need to test for cplusplus here as it is already a cpp source file.

Reply:
Note also that CI build fails with your changes - which kind of means this change isn't compatible with gcc 6?

Reply:
The travis build says that std::exp is not part of std, because math.h does not define it.
I wonder why gcc8 doesn't like the current code

Reply:
That's what I would expect as C functions would not be in a namespace. Could you try using ::exp() and ::log()? Unfortunately I don't have gcc 8.2 to try this out.